subprojects/docs/src/samples/userguide/tutorial/loggingKts/settings.gradle.kts

```diff
@@ -0,0 +1 @@
+rootProject.name = "loggingKts"
```

Comment:
I think it would be better if each variant of a sample had the same Gradle structure, e.g. same project names.

Reply:
👍 to what Adam said above.

Sample directories named something like `logging-groovy` and `logging-kotlin` would make them more obvious to users browsing samples than the cryptic `Kts` suffix.

Reply:
okay